### PR TITLE
Grammatical fixes in the FAQ section

### PR DESCRIPTION
**Pull request by specterflare**
Definitely not all that needs fixing, but all I could do at the moment. I'll make another PR when I edit the rest.

### CODE REVIEW

**Comment by AbdullahM0hamed** [comment]
I see you are a man of culture as well.

File: src/help/faq/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ sidebarDepth: 2
 This can be done by enabling **Download badges** under <Navigation item="library"/>.
 
 ::: guide
-Enable it by going to <Navigation item="library"/> → <Navigation item="filter"/> → **DISPLAY** section then checking the **Download badges** option in the bottom.
+Enable it by going to the <Navigation item="library"/> → <Navigation item="filter"/> → **DISPLAY** section and then checking the **Download badges** option at the bottom.
```
Comment (arkon) [inline src/help/faq/README.md:18]: I feel like it read better without the "the". The other changes are fine though.

File: src/help/faq/README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Refresh your metadata by going to <Navigation item="more"/> → <Navigation item
 :::
 
 #### How do I ignore chapters from certain scanlator groups?
-Every now and then you find a manga that is translated by more than one group, because of this you get multiple releases for every chapter in the manga. To fix this the easiest way is to change sorting to **Total chapters**. You won't see any changes but, when you start reading you'll automatically skip the duplicate releases.
+Every now and then you find a manga that is translated by more than one group; because of this, you get multiple releases for each chapter in the manga. The easiest way to fix this is to change sorting to **Total chapters**. You won't see any changes but, when you start reading you'll automatically skip the duplicate releases.
```
Comment (arkon) [inline src/help/faq/README.md:29]: Re semicolon: might as well just use a period and split it into separate sentences?

File: src/help/faq/README.md [hunk 3]
```diff
@@ -107,14 +107,14 @@ Learn how to create automatic backups [here](/help/guides/creating-backups/#turn
 ### General
 
 #### What are some recommended extensions?
-Due to Covid-19 quarantine, most manga sites are under heavy load. Thus we do not recommend any source at the moment. Also, Tachiyomi is not affiliated with or responsbible for any source that is down or slow.
+Due to the Covid-19 quarantine, most manga sites are under heavy load. Thus we do not recommend any source at the moment. Also, Tachiyomi is not affiliated with or responsible for any source that is down or slow.
```
Comment (arkon) [inline src/help/faq/README.md:110]: Might as well take the opportunity to capitalize COVID-19 properly.

File: src/help/faq/README.md [hunk 4]
```diff
@@ -162,20 +162,20 @@ When installing your first extension and are prompted that your phone isn't allo
 <Carousel name="carousel-unknown-source">
 <CarouselItem>
 <figure class="centered">
-	<h4>Android 8.0 and higher</h4>
-	<p>In newer versions of Android you need to enable "Install unknown apps" by a per-app basis.</p>
-	<video muted loop controls :poster="$withBase('/help/faq/assets/UnknownSources-Android10.png')" height="500" controlslist="nodownload noremoteplayback" preload="none" loading="lazy" crossorigin="use-credentials">
-		<source :src="$withBase('/help/faq/assets/UnknownSources-Android10.webm')" type="video/webm" />
-	</video>
+    <h4>Android 8.0 and higher</h4>
```
Comment (arkon) [inline src/help/faq/README.md:165]: Don't change the indentation -- I'm pretty sure it's supposed to be linted/autofixed?

File: src/help/faq/README.md [hunk 5]
```diff
@@ -185,13 +185,13 @@ If you need more help regarding this, read [this](https://www.theandroidsoul.com
...
 #### How do I uninstall an extension?
-Extensions for **Tachiyomi** are technically installed the same as normal apps. Either you uninstall them through the app section on your phones settings or you go can do it in the app.
+Extensions for **Tachiyomi** are technically installed the same as normal apps. Either you uninstall them through the app section on your phone's settings or you go can do it in the app.
 
 ::: guide
 To uninstall an extension from within **Tachiyomi**, go to <Navigation item="browse"/> → **EXTENSIONS** then tap **Uninstall** on the extension you want to remove.
 :::
 
-#### Why don't I have extensions or any sources in the catalogue?
+#### Why don't I have extensions or any sources in the catalog?
```
Comment (arkon) [inline src/help/faq/README.md:194]: Filthy Americans.

File: src/help/faq/README.md [hunk 6]
```diff
@@ -213,14 +213,14 @@ To uninstall an extension from within **Tachiyomi**, go to <Navigation item="bro
 You do this by following the steps below.
 
 1. Go to <Navigation item="browse"/> → **SOURCES**.
-1. Click on <Navigation item="filter"/> on the top right corner.
-1. Scroll down until you find the language of your source and enable it.
+2. Click on <Navigation item="filter"/> on the top right corner.
+3. Scroll down until you find the language of your source and enable it.
```
Comment (arkon) [inline src/help/faq/README.md:217]: Using `1` for all items is the norm in Markdown.

File: src/help/faq/README.md [hunk 7]
```diff
@@ -315,7 +315,7 @@ To solve this issue, follow the below steps.
 
 #### Source URL has changed
 
-This means that the manga is dead and needs to migrated from **Mangakakalot**/**Manganelo** to different source. Refer to the guide for **Page list is empty** above.
+This means that the manga is dead and needs to migrate from **Mangakakalot**/**Manganelo** to a different source. Refer to the guide for **Page list is empty** above.
```
Comment (arkon) [inline src/help/faq/README.md:318]: "be migrated"?

**Comment by specterflare** [comment]
I've never really used GitHub before so I have zero idea how to act on your suggestions.

**Comment by AbdullahM0hamed** [comment]
Yeah, git isn't all that intuitive for beginners.

Head over to your branch at https://github.com/specterflare/website/tree/patch-4, and edit the files there. They should automatically update here too @specterflare

File: src/help/faq/README.md (hunk 5 above)
Comment (Soitora) [reply src/help/faq/README.md:194]: Instead of renaming this to the inferior version, maybe reword it to better fit the new (not-called-catalogues) system

File: src/help/faq/README.md (hunk 1 above)
Comment (specterflare) [reply src/help/faq/README.md]: Done

File: src/help/faq/README.md (hunk 2 above)
Comment (specterflare) [reply src/help/faq/README.md]: Done

File: src/help/faq/README.md (hunk 3 above)
Comment (specterflare) [reply src/help/faq/README.md]: Done

File: src/help/faq/README.md (hunk 4 above)
Comment (specterflare) [reply src/help/faq/README.md]: Done

File: src/help/faq/README.md (hunk 5 above)
Comment (specterflare) [reply src/help/faq/README.md]: Done

File: src/help/faq/README.md (hunk 6 above)
Comment (specterflare) [reply src/help/faq/README.md]: Done

File: src/help/faq/README.md (hunk 7 above)
Comment (specterflare) [reply src/help/faq/README.md]: Done

**Comment by specterflare** [comment]
Changes have been acted upon
